atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/Transaction.java

```diff
@@ -37,41 +37,123 @@
 
 /**
  * Provides the methods for a transaction with the key-value store.
+ *
+ * In general: users may assume that if maps (including sorted maps) keyed on byte[] are returned to the user,
+ * they may be accessed, including via random access, via any byte array that is equivalent in terms of
```

Comment:
What does "including via random access" mean?

Reply:
Random access here refers to `get(x)` though to be fair the next clause kind of renders this one redundant.